src/commands/help.ts

```diff
@@ -26,20 +26,32 @@ export async function help(context: IActionContext): Promise<void> {
     await selectedItem.handler();
 }
 
+export async function reportIssue(context: IActionContext): Promise<void> {
+    await vscode.commands.executeCommand('workbench.action.openIssueReporter', {
+        extensionId: extensionId,
+        issueBody: undefined, // Leaving repro steps undefined forces the user to type in *something*, which is hopefully helpful
+        data: await getIssueData(),
+    });
+}
+
 async function getStarted(): Promise<void> {
-    /* eslint-disable-next-line @typescript-eslint/no-floating-promises */
-    vscode.env.openExternal(vscode.Uri.parse('https://code.visualstudio.com/docs/containers/overview'));
+    void vscode.env.openExternal(vscode.Uri.parse('https://code.visualstudio.com/docs/containers/overview'));
 }
 
 async function reviewIssues(): Promise<void> {
-    /* eslint-disable-next-line @typescript-eslint/no-floating-promises */
-    vscode.env.openExternal(vscode.Uri.parse('https://github.com/microsoft/vscode-docker/issues'));
+    void vscode.env.openExternal(vscode.Uri.parse('https://github.com/microsoft/vscode-docker/issues'));
 }
 
-async function reportIssue(): Promise<void> {
-    return vscode.commands.executeCommand('vscode.openIssueReporter', `${extensionId}`);
+async function reportIssueFromHelpMenu(): Promise<void> {
+    return vscode.commands.executeCommand('vscode-docker.help.reportIssue');
 }
 
 async function editSettings(): Promise<void> {
     return vscode.commands.executeCommand('workbench.action.openSettings', `@ext:${extensionId}`);
 }
+
+async function getIssueData(): Promise<string> {
```

Comment:
VSCode automatically includes extension version, OS and version, and product name and version, so we don't need to